snippets/epel.erb

```diff
@@ -1,6 +1,10 @@
 <%#
 kind: snippet
 name: epel
-%>
-
-su -c 'rpm -Uvh <%= @host.os.medium_uri(@host, "http://dl.fedoraproject.org/pub/epel/epel-release-latest-#{@host.operatingsystem.major}.noarch.rpm") %>'
+-%>
+<%
+  http_proxy = @host.params['http-proxy'] ? " --httpproxy #{@host.params['http-proxy']}" : nil
+  http_port = @host.params['http-proxy-port'] ? " --httpport #{@host.params['http-proxy-port']}" : nil
+  os_major = @host.operatingsystem.major.to_i
```

Comment:
I just copied it from another template.  Perhaps we need to fix it there too. (I like consistency)
https://github.com/theforeman/community-templates/blob/develop/kickstart/provision.erb#L27